tasks/main.yml

```diff
@@ -5,6 +5,7 @@
 - name: Source specific variables
   include_vars: "{{ item }}"
   with_first_found:
+    - "{{ ansible_os_family }}_{{ ansible_distribution_major_version }}.yml"
```

Comment:
This line becomes superfluous if you follow my recommendation in the vars/RedHat.yml file.

Reply:
Removed as per update.